apps/builder/app/builder/features/props-panel/props-panel.tsx

```diff
@@ -216,16 +255,57 @@ export const PropsPanel = ({
               }}
             />
           )}
-
-          <Button
-            color="neutral"
-            prefix={<PlusIcon />}
-            onClick={() => setAddingProp(true)}
-          >
-            Add property
-          </Button>
-        </div>
+        </Flex>
       </CollapsibleSection>
+      <Separator />
     </Box>
   );
 };
+
+export const PropsPanel = ({
+  selectedInstance: instance,
+  publish,
+}: {
+  publish: Publish;
+  selectedInstance: Instance;
+}) => {
+  const propsMeta = getComponentPropsMeta(instance.component);
+  if (propsMeta === undefined) {
+    throw new Error(`Could not get meta for compoent "${instance.component}"`);
+  }
+
+  const componentMeta = getComponentMeta(instance.component);
+  if (componentMeta === undefined) {
+    throw new Error(`Could not get meta for compoent "${instance.component}"`);
+  }
+
+  const { setProperty: setCssProperty } = useStyleData({
+    selectedInstance: instance,
+    publish,
+  });
+
+  const logic = usePropsLogic({
+    props: useInstanceProps(instance.id),
+    meta: propsMeta,
+    instanceId: instance.id,
+    updateProp: (update) => {
+      store.createTransaction([propsStore], (props) => {
+        props.set(update.id, update);
+      });
+    },
+    deleteProp: (id) => {
+      store.createTransaction([propsStore], (props) => {
+        props.delete(id);
+      });
+    },
+  });
+
+  return (
+    <PropsPanelUI
```

Comment:
Lets call it PropsPanel and instead call  this component PropsPanelContainer or something?

Also this component contains only hooks, does it still make sense to have this component at all if it could be all combined in one hook?

Reply:
> Also this component contains only hooks, does it still make sense to have this component at all if it could be all combined in one hook?

We need `PropsPanelUI` for storybook. If PropsPanelUI would call a hook that contains all logic that would defeat the purpose of separating.

> Lets call it PropsPanel and instead call this component PropsPanelContainer or something?

`PropsPanelContainer` is weird as the name of main exported component that is used in the app.

Reply:
we need a standard convention for this type of components again